web/src/components/FormStateContacts.js

```diff
@@ -119,9 +119,9 @@ const formConfig = {
       zip: '05671-1010'
     },
     medicaidDirector: {
-      name: 'Cory Gustafson',
-      email: 'cory.gustafson@vermont.gov',
-      phone: '802-879‐5901'
+      name: 'First Last',
+      email: 'first.last@vermont.gov',
```

Comment:
Should this be `@state.gov` too, like in `FormStateStart.js`?  (Amusingly, state.gov is still real.)

Reply:
yes, will change